clients/src/test/java/org/apache/kafka/clients/ClientUtilsTest.java

```diff
@@ -102,7 +102,7 @@ public void testResolveUnknownHostException() throws UnknownHostException {
 
     @Test
     public void testResolveDnsLookup() throws UnknownHostException {
-        assertEquals(1, ClientUtils.resolve("localhost", ClientDnsLookup.DEFAULT).size());
+        assertEquals(1, ClientUtils.resolve("kafka.apache.org", ClientDnsLookup.DEFAULT).size());
```

Comment:
Why do we need this change?

Reply:
because `kafka.apache.org` resolves to 2 IP addresses. and I want to be sure that DEFAULT only returns 1 of them. 

`localhost` resolves to 1 IP address.

Reply:
Can you add a comment about this?

Reply:
Added.